python/setup.py

```diff
@@ -8,10 +8,11 @@
     license="Apache License 2.0",
     author="Nipunn Koorapati",
     author_email="nipunn@dropbox.com",
-    py_modules=["mypy_protobuf"],
+    packages=["mypy_protobuf"],
```

Comment:
I don't have any experience making a PyPI package... if there's a better way to do this let me know.

Reply:
Will revert back once the typeshed is updated.